webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/write/PrepareTemplateVariables.scala

```diff
@@ -357,54 +357,9 @@ class PrepareTemplateVariablesImpl(
     // we can't do it when we built node configuration because some (system variable at least (note: but only? If so, we could just check
     // them here, not everything).
 
```

Comment:
all the following checks are now useless. Or more likely: they were already semi-useless, but now, everything is done earlier in `.toPolicy`